client/scraper/youtube_metadata.py

```diff
@@ -32,7 +32,7 @@ def parse_content(self, response):
         media_list = {}
         media_list[FieldKeyword.DATA] = [
             {
-                FieldKeyword.URL: None,
+                FieldKeyword.URL: response.request_url,
```

Comment:
Will we run into a case where we might be nesting multiple youtube requests into one response? I feel like this url should live in prop_map along with all other request/sanitized urls. Cause there's only one oembed link that maps to a particular youtube url